### PR TITLE
Add ability to set tolerations for registry pod

### PR DESCRIPTION
**Pull request by coreydaley**
https://jira.coreos.com/browse/DEVEXP-280

### CODE REVIEW

**Comment by coreydaley** [comment]
/retest 

aws_route.to_nat_gw.3: Error creating route: timeout while waiting for state to become 'success' (timeout: 2m0s)

**Comment by coreydaley** [comment]
/retest

 aws_route.to_nat_gw.3: Error creating route: timeout while waiting for state to become 'success' (timeout: 2m0s)

File: test/e2e/configuration_test.go [hunk 1]
```diff
@@ -73,6 +81,64 @@ func TestPodResourceConfiguration(t *testing.T) {
...
+			},
+			Replicas:    1,
+			Tolerations: tolerations,
+		},
+	}
+	framework.MustDeployImageRegistry(t, client, cr)
+	framework.MustEnsureImageRegistryIsAvailable(t, client)
+	framework.MustEnsureClusterOperatorStatusIsSet(t, client)
+
+	deployments, err := client.Deployments(imageregistryapiv1.ImageRegistryOperatorNamespace).List(metav1.ListOptions{})
```
Comment (dmage) [inline test/e2e/configuration_test.go]: The deployment name is `image-registry`, so you can use Get.

Comment (coreydaley) [reply test/e2e/configuration_test.go]: updated in current push

**Comment by coreydaley** [comment]
@dmage ptal, updated per your comment

**Pull request merged**